### PR TITLE
Unexpected any types replaced

### PR DESCRIPTION
**Pull request by Guzel738**
"Any" types replaced with correct types

### CODE REVIEW

**Comment by Guzel738** [comment]
@soffest thank you, I updated the branch, please review it again

**Comment by whotake** [comment]
@Guzel738 Looks great 👍 Can you also configure Eslint rule `@typescript-eslint/no-explicit-any` to error level? So we don't see unexpected `any` in the future.

**Review by workshur — APPROVED** [review]

File: kafka-ui-react-app/src/serviceWorker.ts [hunk 1]
```diff
@@ -41,6 +41,7 @@ function registerValidSW(swUrl: string, config?: Config) {
               // At this point, the updated precached content has been fetched,
               // but the previous service worker will still serve the older
               // content until all client tabs are closed.
+              // eslint-disable-next-line no-console
```
Comment (whotake) [inline kafka-ui-react-app/src/serviceWorker.ts:44]: @Guzel738 Can you disable `no-console` rule for the whole file instead commenting single lines?

File: kafka-ui-react-app/.eslintrc.json [hunk 2]
```diff
@@ -29,7 +29,8 @@
     "@typescript-eslint/explicit-module-boundary-types": "off",
     "jsx-a11y/label-has-associated-control": "off",
     "import/prefer-default-export": "off",
-    "@typescript-eslint/no-explicit-any": "error"
+    "@typescript-eslint/no-explicit-any": "error",
+    "no-console": "off"
```
Comment (whotake) [inline kafka-ui-react-app/.eslintrc.json:33]: There is no need to disable `no-console` rule for the whole project. We can disable it for `serviceWorker.ts` file.

File: kafka-ui-react-app/.eslintrc.json [hunk 3]
```diff
@@ -28,7 +28,9 @@
     "prettier/prettier": "error",
     "@typescript-eslint/explicit-module-boundary-types": "off",
     "jsx-a11y/label-has-associated-control": "off",
-    "import/prefer-default-export": "off"
+    "import/prefer-default-export": "off",
+    "@typescript-eslint/no-explicit-any": "error",
+    "no-console": "off"
```
Comment (workshur) [inline kafka-ui-react-app/.eslintrc.json:33]: @Guzel738 Could you please remove `no-console` rule from this config?
```suggestion

```

File: kafka-ui-react-app/src/serviceWorker.ts [hunk 4]
```diff
@@ -9,6 +9,7 @@
 
 // To learn more about the benefits of this model and instructions on how to
 // opt-in, read https://bit.ly/CRA-PWA
+/* eslint-disable no-console */
```
Comment (workshur) [inline kafka-ui-react-app/src/serviceWorker.ts:12]: 👍

**Review by whotake — APPROVED** [review]

**Review by workshur — APPROVED** [review]

**Pull request merged**